modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -19,7 +19,7 @@ Support for {op-system} functionality for all IBM Power8 models, IBM Power AC922
 [discrete]
 == Hardware requirements
 
-* Six IBM Power bare metal servers or six LPARs across multiple PowerVM servers
+* Six LPARs across multiple PowerVM servers
```

Comment:
As far as I can tell, this is the first instance of "LPARs" in this assembly. Will most users know what an LPAR is, or do we need to spell it out for the first instance?

Reply:
@lahinson updated as:
```
* Six logical partitions (LPARs) across multiple PowerVM servers
```
fyi: @manojnkumar